### PR TITLE
Docs: Instructions for upgrading sigil from ignore

### PR DESCRIPTION
**Pull request by jaredbeck**
Do I need to run rake or anything after updating docs? I can't remember, and couldn't find anything in CONTRIBUTING.md about it.

### Motivation

If you don't follow these instructions, you are very likely to encounter error 4010

### Test plan

N/A. Docs.

### CODE REVIEW

File: website/docs/static.md [hunk 1]
```diff
@@ -81,6 +81,19 @@ which errors are reported.
...
+
+After changing the sigil of an `ignore`d file, you must run `srb rbi 
+hidden-definitions`.
+
+The last time hidden definitions were generated, the file was `ignore`d, sorbet 
+didn't load it, so anything in it was treated as "hidden" and defined in 
+`sorbet/rbi/hidden-definitions/hidden.rbi`. Now that sorbet
+is allowed to load it, those definitions must be removed from `hidden.rbi`.
+
+If you don't regenerate `hidden.rbi`, you are likely to encounter [error
```
Comment (DarkDimius) [inline website/docs/static.md]:
```suggestion
If you don't regenerate `hidden.rbi`, you could encounter [error 
```

**Comment by DarkDimius** [comment]
Hi Jared, sorry, this fell through the cracks.
Looking to land it.

You'll need to run `yarn prettier` in `website/` to fix formatting

File: website/docs/static.md (hunk 1 above)
Comment (jaredbeck) [reply website/docs/static.md]: The only way you *wouldn't* get an error 4010 is if the previously ignored file was empty, right? So, the word "likely" seems appropriate here.  If I'm right about the probability, I think the docs should express that. Please let me know if I'm wrong.

**Comment by ptarjan** [comment]
ping. Are you interested in landing this still?

**Comment by jaredbeck** [comment]
Thanks for the ping. I'll run `prettier` just a sec

**Comment by jaredbeck** [comment]
> .. I'll run prettier just a sec

run

File: website/docs/static.md [hunk 2]
```diff
@@ -81,6 +81,19 @@ which errors are reported.
...
+After changing the sigil of an `ignore`d file, you must run
+`srb rbi hidden-definitions`.
+
+The last time hidden definitions were generated, the file was `ignore`d, sorbet
+didn't load it, so anything in it was treated as "hidden" and defined in
+`sorbet/rbi/hidden-definitions/hidden.rbi`. Now that sorbet is allowed to load
+it, those definitions must be removed from `hidden.rbi`.
+
+If you don't regenerate `hidden.rbi`, you are likely to encounter
+[error 4010](https://srb.help/4010) (two definitions of the same method).
```
Comment (jez) [inline website/docs/static.md]: static.md is a pretty high level doc (most people reading it don't know what a `typed:` sigil is at all.
On the other hand, this suggestion is really specific to RBI files and specific error messages.

What do you think about moving this suggestion to the error-reference.md suggestion for 4010?
I'm fine linking to the error-reference.md from static.md, but I'd like to keep static.md somewhat short and high level.

Comment (jaredbeck) [reply website/docs/static.md]: > static.md is a pretty high level doc (most people reading it don't know what a typed: sigil is at all.

I like the idea of keeping it high-level, but currently `static.md` is _where_ sigils are introduced. Each is then described in depth. After this, one's next question is "How do I upgrade a sigil?". I've located this new section accordingly.

> What do you think about moving this suggestion to the error-reference.md suggestion for 4010?

What I've written is not just about fixing 4010 (retroactive), it's about how to correctly upgrade a sigil (proactive). They should link to each other though.

> .. I'd like to keep static.md somewhat short and high level.

Maybe `static.md` could cover sigils at a higher level than it currently does, and the in-depth sigil docs could move to a new file, `sigils.md`. The new file could contain the detailed description of each sigil, and the proposed instructions re: upgrading sigils.

**Comment by jaredbeck** [comment]
> Maybe static.md could cover sigils at a higher level than it currently does, and the in-depth sigil docs could move to a new file, sigils.md. The new file could contain the detailed description of each sigil, and the proposed instructions re: upgrading sigils.

Want me to move the detailed stuff to a new file, `sigils.md`, Jake?

**Pull request merged**